students/srepking/Lesson09/donors.py

```diff
@@ -122,19 +116,15 @@ def letters(self):
 
 
 class Individual:
-    def __init__(self):
-        pass
+    def __init__(self, name, donations):
+        self.name = name
+        self.donations = donations
 
-    @staticmethod
-    def thank_you(donor, donation):
+    def add_donation(self, donation):
+        self.donations.append(donation)
+
+    @property
+    def thank_you(self):
         """Add a donation to a donors records and print a report."""
         return ('Thank you so much for the generous gift of ${0:.2f}, {1}!'
-              .format(float(donation), donor))
-
-
-# Method to add a new donor
-
-# Method to search for a given donor
-
-
-# Generate reports about multiple donors
\ No newline at end of file
+                .format(self.donations[-1], self.name))
```

Comment:
when creating objects try to think of all of the properties and features that relate to the object; in this scenario what else belong to single Donor?
* avg donation
* total donations
* sum donations

those can be functions (or even properties if you must)